StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetIntentConfiguration.swift

```diff
@@ -7,10 +7,16 @@
 
 import Foundation
 
-@_spi(STP) public extension PaymentSheet {
+@_spi(ExperimentPaymentSheetDecouplingAPI) public extension PaymentSheet {
     /// 🚧 Under construction
+    /// Contains information needed to render PaymentSheet
+    /// The values are used to calculate the payment methods displayed (if paymentMethodTypes is nil) and influence the UI.
```

Comment:
```suggestion
    /// The values are used to calculate the payment methods displayed and influence the UI.
```

IDK, I'm not sure this adds clarity or is strictly true.